treeherder/perf/alerts.py

```diff
@@ -50,8 +50,7 @@ def generate_new_alerts_in_series(signature):
     # (use whichever is newer)
     max_alert_age = (datetime.now() -
                      settings.PERFHERDER_ALERTS_MAX_AGE)
-    series = PerformanceDatum.objects.filter(signature=signature).filter(
-        push_timestamp__gte=max_alert_age).order_by('push_timestamp')
```

Comment:
I don't think removing this order_by will have a detrimental affect because the series data is used to create an unordered dict - `revision_data` - which is later sorted in `detect_changes`.